### DISCUSSION TITLE
TFT 120 x 160 (Chinese) Blue Background

### DISCUSSION THREAD

**Discussion by CHCHmark** [Bodmer/TFT_eSPI#3536] [General]
Hi guys,
I'm working with a 160 x 120 ST7735 TFT using the eSPI library setup 2. When it powers up I can see the expected code but there is a blue background over everything. I have tried resetting the colour order to no avail. Picture attached
![160x120](https://github.com/user-attachments/assets/705c569b-804d-4e29-a4e0-0a97869078d1)
Have no problems with this library and other displays so it could be a different driver chip on the display.  Any thoughts gratefully received.  Cheers  Mark

**Comment by Basitadam** [comment]
Hi @CHCHmark,

I have not work with this TFT controller but seems there are several color set types. Can you try modifying this  #define line in your setup 2 file:

``` #define ST7735_REDTAB```

with these one by one:

ST7735_INITB
ST7735_GREENTAB
ST7735_GREENTAB2
ST7735_GREENTAB3
ST7735_GREENTAB128
ST7735_GREENTAB160x80
ST7735_ROBOTLCD
ST7735_REDTAB160x80
ST7735_REDTAB
ST7735_BLACKTAB


I suggest power cycling the display after each try

BR,
Tim

**Comment by CHCHmark** [comment]
Hi Tim,

Thank You, that was the answer. I only needed to try ST7735_GREENTAB to sort it.
Appreciate your help.

Cheers

Mark
________________________________

**Reply by Basitadam** [reply]
Wow. That was fast. Glad it helped. HappyFace();
